sdcm/tester.py

```diff
@@ -701,6 +701,25 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
             else:
                 db_info['device_mappings'] = []
 
+        additional_ebs_volumes_num = self.params.get("aws_ebs_volume_num")
+        if additional_ebs_volumes_num > 0:
+            disk_char = "fghijklmnop"
+            for i in range(additional_ebs_volumes_num):
+                ebs_volume = {
+                    "DeviceName": f"/dev/xvd{disk_char[-(i+1)]}",
```

Comment:
Why it reversed?

Reply:
The idea was to safely attach create ebs volume. because drive letter "p" used with less chance than f